### PR TITLE
Add explanation as to why we want feedback

### LINKED ISSUE #311: Add explanation of why we collect feedback

### PR DESCRIPTION
**Pull request by zachlatta**
Closes #311.

@MaxWofford can you review/merge this when you have a chance?

### CODE REVIEW

File: cohort_4/playbook/ACTIVITIES.md [hunk 1]
```diff
@@ -647,6 +647,40 @@ Here are good example failures that haven't been resolved:
...
+2. Actually acting on their feedback and _showing_ that it makes a difference
+
+When it comes time to send out feedback forms to everyone, we recommend saying
+something along the lines of:
+
+> Hey everyone! We're going to send out a link to a feedback form for today. We
+> want to do _everything we can_ to make camp as good as possible and we can't
+> do this without you.
+>
+> Please be as harsh as critical as you feel comfortable, we are going to do
```
Comment (maxwofford) [inline cohort_4/playbook/ACTIVITIES.md]: @zachlatta harsh _AND_ critical

Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: Good catch, will fix when I get back to the office.

Can you ping @gemmabusoni in real life once this is merged? I want to make sure this is accounted for in today's feedback.

## 

Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta

**Comment by maxwofford** [comment]
> 15:20:58 <gemmab> got it! :)

**Comment by zachlatta** [comment]
Fixed the typo.

**Pull request merged**